instrumentation/http/src/main/java/brave/http/HttpRequestParser.java

```diff
@@ -62,8 +62,7 @@
     @Override public void parse(HttpRequest req, TraceContext context, SpanCustomizer span) {
       String name = spanName(req, context);
       if (name != null) span.name(name);
```

Comment:
Think we can remove this null check (it's the one I most wanted to remove since it's the span name :) ).

But if you think we need it I guess we are missing `@Nullable` on the protected `spanName` method.

Reply:
I can fix the lack of nullable